src/main/java/nextstep/subway/domain/Distance.java

```diff
@@ -0,0 +1,57 @@
+package nextstep.subway.domain;
+
+import nextstep.subway.domain.exceptions.NotPositiveNumberException;
+
+import javax.persistence.Column;
+import javax.persistence.Embeddable;
+import java.util.Objects;
+
+@Embeddable
+public class Distance {
+
+    @Column(name = "distance")
+    private int value;
+
+    public static Distance of(int value) {
+        if (value <= 0) {
```

Comment:
```suggestion
        if (value <= MIN_DISTANCE) {
```

매직넘버를 상수로 분리해주면, 가독성이 더 높아질 것 같네요! 👍

Reply:
[c392beb](https://github.com/next-step/atdd-subway-fare/pull/289/commits/c392beb786c6fcdf78f063dbf9c73887dd29d260) 매직넘버 제거하였습니다! 👍